Dockerfile

```diff
@@ -1,24 +1,20 @@
-# Build image
-FROM golang:1.20-alpine as build
+FROM golang:1.21-bullseye as build
```

Comment:
swap this to bookworm imo

Reply:
I wanted to, but distroless doesn't have a Bookworm image yet.